backend/src/server.js

```diff
@@ -2,7 +2,27 @@ const dotenv = require('dotenv').config();
 const express = require("express");
 const Alby = require("ably");
 const p2 = require("p2");
+const path = require("path");
+const cors = require("cors");
 
 const app = express();
 
-console.log("hello world!");
\ No newline at end of file
+app.listen(3000);
+
+app.use(express.urlencoded({ extended: true }));
+app.use(express.static('public'));
+app.use(express.json());
+app.use(cors());
+  
+// tu pozniej to przeslac do bazy danych
```

Comment:
you change this comment `// TODO:` with description of what should be done (and it would be best to include number of ticket in which this should be done)